api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/organizations_controller.rb

```diff
@@ -211,12 +215,14 @@ def batch_import
 
   protected
 
-  def organization_permitted_update_params
-    [:name, :billing_currency]
+  def organization_params
+    params.require(:organization)
   end
 
   def organization_update_params
-    params.fetch(:organization, {}).permit(*organization_permitted_update_params)
+    organization_params.permit(*ATTRIBUTES).tap do |whitelisted|
```

Comment:
For backward compatibility (as we might have some projects overriding `organization_permitted_update_params`) could we keep:
```ruby
def organization_permitted_update_params
  ATTRIBUTES
end

def organization_update_params
  organization_params.permit(*organization_permitted_update_params).tap do |whitelisted|
    ...
  end
end
```

`ATTRIBUTES` could be renamed to `UPDATEABLE_ATTRIBUTES` (or `WRITEABLE`) for clarity